tools/mo/openvino/tools/mo/front/mxnet/extractors/batch_dot_ext.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+from openvino.tools.mo.front.extractor import FrontExtractorOp
+from openvino.tools.mo.front.mxnet.extractors.utils import get_mxnet_layer_attrs
+from openvino.tools.mo.graph.graph import Node
+from openvino.tools.mo.ops.MatMul import MatMul
+
+
+class BatchDotExt(FrontExtractorOp):
+    """
+    MXNet operation which compute dot product of x and y, where x and y are data in batch: [B_0, B_1, ..., B_N, :, :].
```

Comment:
If this operation is equivalent to `MatMul` operation, then the term `dot product` is incorrect, because `dot product` means that **only one dimension** of `x` and **only one dimension** of `y` are used to perform multiplication.